TRACKPAD_SCROLL_FIX.md

```diff
@@ -0,0 +1,129 @@
+# Trackpad Scroll Speed Fix
```

Comment:
This isn't an appropriate file to add to the repository; it's just information about this PR.